test/new-e2e/tests/containers/k8s_test.go

```diff
@@ -118,7 +118,7 @@ func (suite *k8sSuite) TestAgent() {
 					}
 				}
 			}
-		}, 2*time.Minute, 10*time.Second, "Not all agents eventually became ready in time.")
```

Comment:
Timeout is also increased here. Does k8s_test have similar issue now?

Reply:
I recently added the retry loop around the tests of Kubernetes pod readiness in #20302.
I originally put 2 minutes because the tests were sometimes passing. So, it meant that, when it was failing, the pods were “most probably” close to be ready and that we shouldn’t have to wait too long.
It turns out that, since #20302, I feel that failures of pod readiness test are more rare. And also, they now affect only Windows pods. Whereas, prior to #20302, it was also affecting Linux pods.
So, I guess Windows nodes are taking more time to pull the Windows agent image and to start it.
That’s why I also bumped the timeout here.